docker/dev/xain-fl.toml

```diff
@@ -0,0 +1,29 @@
+[server]
```

Comment:
Configuration files do not belong to docker, would be great to move them to dedicated folder with configurations.
On the side note: as we are not going to have any other configurations except dev and example published in open source, make sense not to create a dedicated folder to the only configuration we have, what do you think?

Reply:
Some parts of the config are specific to docker image we build. For example the path to the file that contains the initial weights is `/initial_weights.npy` in docker, but will most likely differ from one environment to another. That's why I put them in the `docker/` directory. I'm open to moving them, let me know if you prefer.

Reply:
I moved the config file to the top of the repo and remove `docker/dev`